### PR TITLE
CLI: Updated docs for the create table command

### PR DESCRIPTION
**Pull request by hosekpeter**
Jira issue(s): [PROOF-155](https://keboola.atlassian.net/browse/PROOF-155)

Changes:

- updated flags for create table command
- updated example for a defining column types

---

[PROOF-155]: https://keboola.atlassian.net/browse/PROOF-155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hhanova — APPROVED** [review]

File: cli/commands/remote/create/table/index.md [hunk 1]
```diff
@@ -6,29 +6,27 @@ permalink: /cli/commands/remote/create/table/
...
-`--name <string>`
-: Name of the table
-
-`--primary-key <string>`
-: Comma-separated list of columns used as a primary key
+- `--bucket <string>`: Specifies the bucket ID where the table will be created.
+- `--columns <string>`: Defines a comma-separated list of column names for the table.
+- `--columns-from <string>`: Indicates the path to the table definition file in JSON format.
+- `--name <string>`: Sets the name of the new table.
+- `--primary-key <string>`: Determines a comma-separated list of columns to be used as the primary key.
```
Comment (michaljurecko) [inline cli/commands/remote/create/table/index.md:23]: @hhanova hi, this your change breaks the visual aspect of the page and readability

-------------------

Before: 
image


--------------

After:

image

File: cli/commands/remote/create/table/index.md [hunk 2]
```diff
@@ -6,29 +6,27 @@ permalink: /cli/commands/remote/create/table/
 * TOC
 {:toc}
 
-**Create a [table](https://help.keboola.com/storage/tables/) in Keboola Storage.**
+## Create a Table from the CLI
+
```
Comment (michaljurecko) [inline cli/commands/remote/create/table/index.md:10]: Why is this needed?

image

File: cli/commands/remote/create/table/index.md [hunk 3]
```diff
@@ -61,15 +56,19 @@ Enter a comma-separated list of column names.
...
 ```
-To define column types, choose `y/Y`, then start an editor. 
+**Defining column types:**
+
+To define column types, select `y/Y`. Then, start an editor. 
 
 ```
-Would you define column types?
+Want to define column types?
 ? Columns Types Definition: Yes
```
Comment (michaljurecko) [inline cli/commands/remote/create/table/index.md:66]: @hosekpeter you should propagate this change to the CLI code/tests.

File: cli/commands/remote/create/table/index.md (hunk 1 above)
Comment (hhanova) [reply cli/commands/remote/create/table/index.md:23]: I don't agree that these changes break readability... It's just different :-).

For now, I'll leave it the way the other command sections were done (didn't know about that; see the comment below).

File: cli/commands/remote/create/table/index.md (hunk 2 above)
Comment (michaljurecko) [reply cli/commands/remote/create/table/index.md]: > I automatically did it the same way we do it in the user docs for consistency.

I still don't understand two almost identical headings below each other, I would understand the subheading: `Into` or `Overview`.

File: cli/commands/remote/create/table/index.md (hunk 1 above)
Comment (michaljurecko) [reply cli/commands/remote/create/table/index.md]: > I was not asked to proofread the other (older) commands in the developer docs, so I had no idea they were done differently. Sorry about that. I should have looked.

You were :slightly_smiling_face:  you did a review on it, but we haven't changed anything there for a long time.

Ok
- It is now not consistent with the other CLI commands (the main issue).
- Elsewhere in the documentation, we don't have CLI flags, so it's probably hard to compare it with something else.
- You are primarily interested in the names of the flags, description is secondary, that reflects the original design.
- The CLI will probably have a separate documentation page, automatically generated.
- So we will revert the list of flags back. Thanks!

Comment (hhanova) [reply cli/commands/remote/create/table/index.md]: I reversed the changes you did not like. The descriptions were still secondary, but I understand it should be consistent with the other pages. 

As for me doing a review of other commands... You might be right. I could have just forgotten about that :-). But that does not change the fact that there are command pages that have not been proofread. I'll do it asap.
